index.html

```diff
@@ -33,9 +33,149 @@
      <a href="https://www.google.com"><i class="fa fa-google"></i></a>
     </li>
     </ul>
+   
+        <img class="seebutton" src="./Disabled.png" alt="button">
+    
 </div>
 </section>
-
+<section>
+    <div class="second-header">
+        <h2 class="second">My Recent work</h2>
+        <img src="./Indicator.png" alt="indicator">
+    </div>
+    <div class="grid-container">
+        <div class="grid-item first">
+            
+           <div class="item image"></div> 
+           <h2 class="title">Multi-Post Stories Gain + Glory</h2>
+           <div class="tagcontainer">
+            <label>Ruby on Rails</label>
+            <label>Javascript</label>
+            <label>CSS</label>
+            <label>HTML</label>
+           </div>
+           <button class="see-post"><p class="see-button">see project</p></button>
+           </div>
+       
+        <div class="grid-item third">
+            <div class="item image"></div> 
+           <h2 class="title">Multi-Post Stories Gain + Glory</h2>
+           <div class="tagcontainer">
+            <label>Ruby on Rails</label>
+            <label>Javascript</label>
+            <label>CSS</label>
+            <label>HTML</label>
+           </div>
+           <button class="see-post"><p class="see-button">see project</p></button>
```

Comment:
- [ ] your button design doesn't match the given figma template please make sure to check the [figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=48-988&t=96rIVjVf1SD2o0ys-0) and make the correct changes. it is important to show recruiters that you pay attention to details when you completely provide the designs as shown on figma. :+1: 
- figma design
![btnn-f](https://user-images.githubusercontent.com/89055502/235299310-bd0e5250-4443-4f87-98f0-253f90e56118.png)
- your design
![btn-s-12](https://user-images.githubusercontent.com/89055502/235299320-000a004b-b0ae-4f93-b4a4-5074b8d2a6f1.png)